.circleci/config.yml

```diff
@@ -105,11 +107,14 @@ jobs:
           path: /tmp/artifacts/
   build-docker-image:
     docker:
-      - image: circleci/buildpack-deps:xenial
+      - image: circleci/node:8
```

Comment:
This is to make sure npm is available in this step and is consistent with the other steps where bundles are collected.